tests/docker/Dockerfile

```diff
@@ -153,4 +153,4 @@ RUN useradd -u $UID -ms /bin/bash ducker \
 
 USER ducker
 
-CMD sudo service ssh start && tail -f /dev/null
```

Comment:
we use the `tail` to make the container alive, and that can be replaced by detaching the sshd. Hence, could you please try to use `CMD ["sudo", "service", "ssh", "start", "-D"]` instaead?